src/@resources/deployment/inputs.ts

```diff
@@ -155,6 +155,11 @@ export type DeploymentInputs = {
     target_port: number;
   }[];
 
+  /**
+   * Target ports for services of the deployment
+   */
+  service_ports?: number[];
```

Comment:
I'd recommend you update the resource type to match this PR. Since we already have the "services" array, it would be perfect to add a field to it (that we can load the value of automatically) instead of creating another array:

https://github.com/architect-team/arcctl/pull/99/files#diff-0924311dae076425a5cbb1a6bd35f1864fde14f752d7b1bd68d8bfb730d96f32

Reply:
Good call - do you mean here: https://github.com/architect-team/arcctl/pull/98/files#diff-bc589e1f58770257ed1c2691c62121f9cca831d53268d911040405e46c4f54ffR309-R312 or instead using the `exposed_ports` (which sounded like perhaps something different)?